lib/Models/Catalog/Ckan/CkanCatalogGroup.ts

```diff
@@ -261,6 +261,15 @@ export class CkanServerStratum extends LoadableStratum(CkanCatalogGroupTraits) {
       return;
     }
 
+    // Return if excluding inactive dataset and dataset is inactive
+    if (
```

Comment:
Is it possible to have a case where only one of `state` or `data_state` is active and that means it's still a valid working datasource? Like state=active and dataState=inactive? Probably not just checking.

Also, is the only allowable value for `dataState` or `state` "active"? Is this a ckan standard? I see in the docs:

"state (string) – the current state of the dataset, e.g. 'active' or 'deleted', only active datasets show up in search results and other lists of datasets, this parameter will be ignored if you are not authorized to change the state of the dataset (optional, default: 'active')"

But it does say "e.g. 'active' or 'deleted'" implying that other values can be used here... Is it possible we need to account for a wider range of values? Current logic just says that if the flag is present, and it has any other value than active, dont show it... Reckon thats OK?

Reply:
From flicking through [ckan](https://github.com/ckan/ckan) code base - I can see the following values for `state`
- "active" 
- "deleted" 
- "draft"

As for `data_state` - we have the schema here https://github.com/datagovau/ckanext-datagovau/blob/966e045f8bfe0cc1453d9eaf6511e8c476d59c26/ckanext/datagovau/schemas/dataset.yaml#L91-L100

- "active" 
- "inactive" 

As for why data.gov.au has an additional property - I have no idea. All of the datasets we harvest through data.gov.au have `state = "active"`.

I have added some comments to clarify both properties. 

https://github.com/TerriaJS/terriajs/blob/9f9b414d1f7149eadcdbe39f8842378404a341de/lib/Models/Catalog/Ckan/CkanDefinitions.ts#L55-L62

I have changed logic to check for values other than `"active"`:

https://github.com/TerriaJS/terriajs/blob/9f9b414d1f7149eadcdbe39f8842378404a341de/lib/Models/Catalog/Ckan/CkanCatalogGroup.ts#L258-L268

Reply:
Oh yeah that looks solid!